src/menu_new_game/A5CC50.c

```diff
@@ -0,0 +1,336 @@
+    viSetScreenOffsets(sp24, sp24, sp20, sp20);
+    func_800067DC();
+    while (1) {
+        ohWait(1);
+    }
+}
+
+void func_800E2054_A5D404(Gfx** gfxPtr) {
+    gSPDisplayList((*gfxPtr)++, D_800E35B0);
+}
+
+void new_game_init(void) {
+    func_8036A3F8_83DBA8(D_80118118, sizeof(D_80118118));
+    func_800AAE28();
+    func_800AA85C(24, 6);
+    func_800AA870(0xF0000);
+    func_8036EB98();
+    ohCreateCameraWrapper(0, 0x80000000, 100, 6, 0);
+    D_801180B0 = ohCreateCamera(3, ohUpdateDefault, 0, 0x80000000, renSpriteCameraRender, 3, 2, -1, TRUE, 1, NULL, 1, TRUE);
+    D_801180B0->data.cam->flags = 8;
+    func_800A7F68(0, 0x101);
+    omCreateProcess(omAddGObj(0xE, NULL, 0, 0x80000000), func_800E1F58_A5D308, 0, 1);
+}
+
+SceneSetup menu_new_game_setup = {
+    {
+        0,                          /* unk_00*/
+        omUpdateAll,                /* fnUpdate */
+        omDrawAll,                  /* fnDraw */
+        menu_new_game_VRAM_END,     /* heapBase */
+        0,                          /* heapSize */
+        2,                          /* unk14 */
+        1,                          /* numContexts */
+        0x5000,                     /* dlBufferSize0 */
+        0x2000,                     /* dlBufferSize1 */
+        0x0000,                     /* dlBufferSize2 */
+        0x0000,                     /* dlBufferSize3 */
+        0xC800,                     /* gfxHeapSize */
+        0,                          /* unk30 */
+        0x2000,                     /* rdpOutputBufferSize */
+        func_800A1A50,              /* fnPreRender */
+        contUpdate                  /* fnUpdateInput */
+    },
+    32,                             /* numOMThreads */
+    1024,                           /* omThreadStackSize */
+    128,                            /* numOMStacks */
+    0,                              /* unk4C */
+    64,                             /* numOMProcesses */
+    64,                             /* numOMGobjs */
+    0x5C,                           /* objectSize */
+    1024,                           /* numOMMtx */
+    0,                              /* unk60 */
+    NULL,                           /* unk64 */
+    16,                             /* numOMAobjs */
+    8,                              /* numOMMobjs */
+    16,                             /* numOMDobjs */
+    0x88,                           /* omDobjSize */
+    128,                            /* numOMSobjs */
+    0x58,                           /* omSobjSize */
+    8,                              /* numOMCameras */
+    0x90,                           /* omCameraSize */
+    new_game_init                   /* postInitFunc */
+};
+
+s32 menu_new_game_entry(s32 arg0) {
+    viApplyScreenSettings(&menu_new_game_video_settings);
+    menu_new_game_setup.gtlSetup.heapSize = D_80369F80 - menu_new_game_VRAM_END;
+    gtlDisableNearClipping(1);
+    omSetupScene(&menu_new_game_setup);
+    func_800BFEBC_5CD5C(16, 1);
+    return 11;
```

Comment:
Should we add GAMESTATE_* constants for these? (Or SCENE_*, to match some other terms used?)

Reply:
yeah, I can add them